core/domain/classifier_domain.py

```diff
@@ -162,6 +162,5 @@ def validate(self):
             raise utils.ValidationError(
                 'Expected cached_classifier_data to be a dict, received %s' %(
                     self.cached_classifier_data))
-        classifier_class = classifier_registry.Registry.get_classifier_by_id(
-            self.algorithm_id)
+        classifier_class = classifier_registry.Registry.get_classifier_by_algorithm_id(self.algorithm_id) # pylint: disable=line-too-long
```

Comment:
We want to keep as many of the pylint rules as possible, in cases where the line is too long, please adopt the approach as used in `core/domain/classifier_registry_test.py` line 28-29.